tests/components/netatmo/test_data_handler.py

```diff
@@ -0,0 +1,101 @@
+"""The tests for Netatmo data handler."""
+from time import time
+from unittest.mock import patch
+
+from homeassistant.components.netatmo import DOMAIN
+from homeassistant.components.netatmo.data_handler import (
+    CAMERA_DATA_CLASS_NAME,
+    NetatmoDataHandler,
+)
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+
+
+async def test_data_handler(hass, config_entry):
```

Comment:
This is testing an integration detail, which we shouldn't do.

Reply:
I totally agree. That's why I pushed it back to see whether it will be necessary once all other Netatmo test PRs are done.